esp/esp/program/modules/apps.py

```diff
@@ -2,3 +2,9 @@
 
 class ModulesConfig(InstallConfig):
     name = 'esp.program.modules'
+
+    def ready(self):
+        super(ModulesConfig, self).ready()
+        # TODO(benkraft): add a thing to InstallConfig that imports a signals
```

Comment:
Whoops, I had it in my head to do that, too, and then forgot.  Fixed both issues.